226. Invert Binary Tree.py

```diff
@@ -0,0 +1,25 @@
+# Definition for a binary tree node.
+# class TreeNode:
+#     def __init__(self, x):
+#         self.val = x
+#         self.left = None
+#         self.right = None
+
+class Solution:
+    def invertTree(self, root):
+        """
+        :type root: TreeNode
+        :rtype: TreeNode
+        """
+        def invert(root):
+            if not root:
+                return root
+            tmp = root.left
```

Comment:
in python u can do multi assignment:
`root.left, root.right = root.right, root.left`